packages/trader/src/AppV2/Components/TradeParameters/Multiplier/multiplier.tsx

```diff
@@ -1,24 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
 import clsx from 'clsx';
 import { observer } from 'mobx-react';
-import { TextField } from '@deriv-com/quill-ui';
+import { ActionSheet, Skeleton, TextField } from '@deriv-com/quill-ui';
 import { Localize } from '@deriv/translations';
 import { useTraderStore } from 'Stores/useTraderStores';
+import Carousel from 'AppV2/Components/Carousel';
+import CarouselHeader from 'AppV2/Components/Carousel/carousel-header';
+import TradeParamDefinition from 'AppV2/Components/TradeParamDefinition';
+import MultiplierWheelPicker from './multiplier-wheel-picker';
 
 type TMultiplierProps = {
     is_minimized?: boolean;
 };
 
 const Multiplier = observer(({ is_minimized }: TMultiplierProps) => {
-    const { multiplier } = useTraderStore();
+    const { multiplier, multiplier_range_list, commission, onChange, currency } = useTraderStore();
+
+    const [isOpen, setIsOpen] = useState(false);
+    const is_small_screen_device = window.innerHeight <= 640;
```

Comment:
hm, interesting, has a feeling that I saw it somewhere before)

Reply:
you are dreaming code in your sleep 😅